src/store/actions.js

```diff
@@ -19,11 +21,34 @@ export default {
     commit('setNodeStatus', 'connecting');
     if (process.env.IS_EXTENSION) postMessage({ type: AEX2_METHODS.SWITCH_NETWORK, payload });
   },
-  async fetchTransactions({ state, getters }, { limit, page, recent }) {
+  async fetchPendingTransactions({
+    state: {
+      sdk,
+      account: { publicKey },
+    },
+  }) {
+    if (sdk.then) await sdk;
+    const txs = await Promise.all(
```

Comment:
It looks like that you are passing an array of transactions instead of an array of promises to `Promise.all `. It will do nothing in this case.